test/integration/generated/paging/src/models/mappers.ts

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ * Changes may cause incorrect behavior and will be lost if the code is regenerated.
+ */
+
+import * as coreHttp from "@azure/core-http";
+
+export const ProductResultValue: coreHttp.CompositeMapper = {
```

Comment:
You know...I wish you were pushing forward more the idea of just generating code to do this rather than generating models...

Reply:
Someday Richard...someday...